manifest.yaml

```diff
@@ -1,65 +1,82 @@
-# directory as this manifest.yaml
-has-instructions: true
-
-# Denotes the version required and recommended
-# Required means that the package will not install if unsatisfied
-# Recommended means that the package will not be listed by the registry to instances of EmbassyOS that do not satisfy it
-os-version-required: "^0.2.13"
-os-version-recommended: "^0.2.13"
-
-# Port mappings from tor ports to the container port
-ports:
-  - internal: 3000
-    tor: 80
-    # if lan is standard, it will proxy HTTP traffic from 443 to the internal port
-    lan: standard
-
-# Do not change this
-image:
-  type: tar
-
-# specifies where on the service's file system its persistence directory should be mounted prior to service startup
-mount: /root
-
-# Directory you would like to expose to dependent services, specified relative to the root of the persistence directory.
-# This will be a read-only filesystem for the dependent service
-public: public
-
-# Directory you would like to expose to dependent services, specified relative to the root of the persistence directory.
-# This will be a read-write filesystem for the dependent service
-shared: shared
-
-# Hidden Service version you would like to use for your services .onion url. Allowed options are "v2" and "v3"
-hidden-service-version: v3
-
-# This is a key value map specifying dependency services that you need to be able to function
-# The keys are the service id's on which you depend.
-# NOTE: if you are developing a standalone service, you may delete this entire stanza
+assets:
+  license: LICENSE
+  icon: icon.png
+  instructions: instructions.md
+  docker-images: image.tar
+  icon-type: png
+main:
+  type: docker
+  image: main
+  entrypoint: "/usr/local/bin/docker_entrypoint.sh"
+  args: []
+  mounts: {}
+  io-format: yaml
+health-checks: {}
+config:
+  get:
+    type: docker
+    image: compat
+    system: true
+    entrypoint: config
+    args: 
+      - "get"
+      - "/root"
+    mounts: {}
+    io-format: yaml
+  set:
+    type: docker
+    image: compat
+    system: true
+    entrypoint: config
+    args:
+      - "set"
+      - "/root"
+    mounts: {}
+    io-format: yaml
+volumes: {}
+interfaces:
+  main:
+    tor-config:
+      port-mapping:
+        3000: "80"
+    ui: true
+    protocols:
+      - http
+backup:
+  create:
+    type: docker
+    image: compat
+    system: true
+    entrypoint: "true"
+    args: []
+    mounts: {}
+  restore:
+    type: docker
+    image: compat
+    system: true
+    entrypoint: "true"
+    args: []
+    mounts: {} 
 dependencies:
   lnd:
-    # The version range that is acceptable for this dependency
     version: ">=0.12.1 <0.14.0"
-    # Description of the dependency relationship
     decription: Thunderhub is an LND Manager
-    # If set to true it will mount the public directory of the dependency to $mount/start9/public/$dependencyId
     mount-public: true
```

Comment:
`mount-public` and `mount-shared` are no longer keys. `main.mounts` is where the new mounted directories are contained, [wip example](https://github.com/Start9Labs/btcpayserver-wrapper/pull/7/files#diff-d865b06b497efc8c56c63e5bedfc86f6da6a005cdb7d30e3702286f3d918aaa6R28)